persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageMapper.kt

```diff
@@ -487,6 +490,37 @@ object MessageMapper {
                 restrictedAssetName.requireField("assetName")
             )
 
+            MessageEntity.ContentType.COMPOSITE -> {
+                // if the text body is null then the composite message had no text body
```

Comment:
maybe a local variable stating the above instead of a comment would be better here ?

Reply:
there is already a local val
`                val compositeText: MessageEntityContent.Text? = text?.let {
`

the comment just so no body add a `.requireField` to the text content since it is fine if it is missing